src/Angor/Client/Pages/View.razor

```diff
@@ -162,43 +163,60 @@
         </div>
 
         <!-- Stages Section -->
-
-
-        <div class="card card-body mt-4">
-            <div class="d-flex align-items-center">
-                <span class="user-select-none">
-                    <Icon IconName="stages"></Icon>
-                </span>
-                <div class="h-100 ms-3">
-                    <h5 class="mb-0 font-weight-bolder">
-                        Stages
-                    </h5>
+       
+            <div class="card-body mt-4">
```

Comment:
this are UI changes that might effect what @miladsoft just did can you check this Milad?